ftw/news/browser/news_listing_block.py

```diff
@@ -74,6 +74,10 @@ def get_item_dict(self, brain):
         if utils.can_view_about():
             author = utils.get_creator(obj)
 
+        image_tag = ''
+        if self.context.show_lead_image:
```

Comment:
Adapting the context is the safe and standard way to access a field value in dexterity:
```python
if INewsListingBaseSchema(self.context).show_lead_image:
  ...
```